src/spans/awsSpan.js

```diff
@@ -270,15 +273,17 @@ export const getHttpSpan = (randomRequestId, requestData, responseData = null) =
   const { awsServiceData, spanId } = serviceData;
 
   const prioritizedSpanId = getHttpSpanId(randomRequestId, spanId);
-  let httpInfo = {
-    host: requestData.host,
-    request: payloadStringify(requestData),
-    response: payloadStringify(responseData),
-  };
```

Comment:
I moved it to be in the `catch` to improve performance (not related to my ticket), not need to to run `payloadStringify` on the http data twice.